requirements/dev.txt

```diff
@@ -1,2 +1,2 @@
-https://github.com/celery/py-amqp/zipball/master
-https://github.com/celery/vine/zipball/master
+https://github.com/celery/py-amqp/zipball/v2.6
```

Comment:
This should not be changed at all.

Reply:
I wasn't able to build locally using tox without this, if I remember correctly.

Reply:
Reverting since unittest is failing on latest commit.